support-frontend/assets/pages/[countryGroupId]/components/formOnSubmit.ts

```diff
@@ -106,6 +91,22 @@ export const submitForm = async ({
 	const supportAbTests = getSupportAbTests(abParticipations);
 	const deliveryInstructions = formData.get('deliveryInstructions') as string;
 	const similarProductsConsent = getConsentValue(formData, CONSENT_ID);
+	const deliveryAgent =
+		productFields.productType === 'Paper'
+			? productFields.deliveryAgent
+			: undefined;
```

Comment:
I wonder if there's an argument that `buildProductInformation` should be doing this. What do you think?

Reply:
Yup 👍